frontend/src/Home.js

```diff
@@ -45,7 +45,7 @@ export const Home = () => (
         <FilesToUpload path="/p1/uploadfilesquestion" />
         <FileUploadPage path="/p1/uploadfiles" />
         <Willbecontacted path="/p1/contactinfoquestion" />
-        <ContactInfoPage path="/p1/contactinfo" />
+        <ContactInfoPage path="/p2/contactinfo" />
```

Comment:
This will get rid of Prototype 1's contactinfo page - instead we want to do something like
```
 import { ContactInfoPage as P2ContactInfoPage } from './p2/ContactInfoPage'
```
above (leaving the p1 import alone) and here add
```
<P2ContactInfoPage path="/p2/contactinfo" />
```
while leaving `/p1/contactinfo` in place.